docs/docker.md

```diff
@@ -0,0 +1,32 @@
+
+## Prerequisites
+1. Docker installed  [(Docker for Mac, Docker for windows, linux, etc)](https://www.docker.com/get-docker)
+
+## Steps to run the git repo unittests from docker
+
+1.  Build the Dockerfile. `` docker build -t moviepy -f Dockerfile . ``
+2.  Get a bash prompt in the moviepy container 
+
+```
+cd tests
+docker run -it -v `pwd`:/tests moviepy bash
+```
+
+3. run the tests.. `` python test_issues.py ``
```

Comment:
since the tests are going to be mounted into /tests, you will need to ``cd tests`` before the ``python test_issues.py``